src/AppBundle/Controller/ClassesController.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Classes;
+use AppBundle\Entity\Teachers;
+use AppBundle\Repository\ClassesRepository;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TelType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\HttpFoundation\Request;
+
+class ClassesController extends Controller
+{
+    /**
+     * @Route("classes", name="classes_listing")
+     */
+    public function indexAction()
+    {
+        $classes = $this->getDoctrine()
+            ->getRepository(Classes::class)
+            ->getClassesWithTeacher();
+        return $this->render('classes/index.html.twig', [
+            'classes' => $classes,
+        ]);
+    }
+
+    /**
+     * @Route("class/create", name="classes_create")
+     */
+    public function createAction(Request $request)
+    {
+        $classes = new Classes();
+
+        $teachers = $this->getDoctrine()
```

Comment:
Create a service and move this logic to that service, keep you controllers as thin as possible.